internal/pkg/db/db_test.go

```diff
@@ -56,6 +56,75 @@ func (dbs *dbSuite) TearDownSuite() {
 	dbs.Backend.CloseClient()
 }
 
+func (dbs *dbSuite) TestAddDocumentRevision() {
+	baseUUID := dbs.documents[1].GetMetadata().GetId()
+	revisionUUID := dbs.documents[0].GetMetadata().GetId()
+
+	tempbe, err := testutil.NewTestBackend()
+	dbs.Require().NoError(err, "failed database backend creation")
+	defer tempbe.CloseClient()
```

Comment:
Might be good candidates for `SetupSubTest` and `TearDownSubTest`

Reply:
I'm going to look more into this. The tests in this file specifically expect a pristine db, in a state after SetupSuite(), which is how it should be, but me monkeying around with annotations and aliases to test them is impacting others, so I'll figure out a way to keep everything tidy

Reply:
Ending up using Setup/Teardown Subtest functions, but I moved my test additions to a different test file, since they are the only tests making annotation/alias adjustments. lmk what you think